authentication.go

```diff
@@ -134,7 +135,7 @@ func getUnlockingScript(tx *bt.Tx, inputIndex uint32, privateKey *bec.PrivateKey
 func createSignature(xPriv *bip32.ExtendedKey, bodyString string) (payload *models.AuthPayload, err error) {
 	// No key?
 	if xPriv == nil {
-		err = apierrors.ErrMissingXPriv
+		err = CreateErrorResponse("error-unauthorized-missing-xpriv", "missing xpriv")
```

Comment:
would not be better to create the error kind in `errors.go` and refer it here ?

Reply:
Done